src/extension.ts

```diff
@@ -47,7 +47,8 @@ function updateConfigs(status:vscode.StatusBarItem) {
     let config = vscode.workspace.getConfiguration("branchwarnings");
     protectedBranches = config.get<string[]>("protectedBranches");
     suppressPopup = config.get<boolean>("suppressPopup");
-    status.color = config.get<string>("msgColor");
+    status.backgroundColor = new vscode.ThemeColor('statusBarItem.errorBackground');
+    status.command = "git.checkout";
```

Comment:
This command mimics when you click the `master` branch status bar button to start a branch right from the warning.